public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -426,6 +438,7 @@ function AddAnomalyDetector({
         initialValues={initialDetectorValue}
         onSubmit={handleSubmit}
         validateOnChange={true}
+        validate={validateFeatures}
```

Comment:
What is this function doing and why do we need it? Isn't the feature validation covered in `handleValidationAndSubmit` like you have above?

Reply:
this is an existing validation function checks on feature form. It will be called at `formikProps.validateForm()` in `handleValidationAndSubmit`